### PR TITLE
Metadata cleanup

### PR DESCRIPTION
**Pull request by vsedmik**
Updated CaseComponents, CaseLevels, CaseImportance and tiers to match the new metadata guidelines.

### CODE REVIEW

File: tests/foreman/cli/test_host.py [hunk 1]
```diff
@@ -741,6 +743,7 @@ def test_negative_register_twice(self):
             # host being already registered.
             self.assertEqual(result.return_code, 64)
 
+    @tier3
```
Comment (mirzal) [inline tests/foreman/cli/test_host.py]: flake on travis complains, that this decorator is not imported.

File: tests/foreman/ui/test_jobinvocation.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 :CaseLevel: Acceptance
 
-:CaseComponent: UI
+:CaseComponent: WebUI
```
Comment (mirzal) [inline tests/foreman/ui/test_jobinvocation.py]: Earlier comment about interface component applies.

I think this one should be `RemoteExecution`? Seems to be closest match here. But you can double-check with someone more familiar with the feature.

File: tests/foreman/sys/test_rename.py [hunk 3]
```diff
@@ -1,13 +1,13 @@
 # -*- encoding: utf-8 -*-
 """Test class for ``katello-change-hostname``
 
-:Requirement: katello-change-hostname
+:Requirement: satellite-change-hostname
```
[the remark this answers is not in the document]
Comment (vsedmik) [reply tests/foreman/sys/test_rename.py]: Thank you for pointing out, fixed.

File: tests/foreman/ui/test_host.py [hunk 4]
```diff
@@ -385,7 +385,7 @@ def module_libvirt_hostgroup(
     ).create()
 
 
-@tier3
+@tier4
```
[the remark this answers is not in the document]
Comment (vsedmik) [reply tests/foreman/ui/test_host.py]: Here I just followed the documentation. These tests (decorated by tier4) took from 15 to 42 minutes to complete each (average is around 25 minutes), based on the report portal launches #146 and #144 data, so I thought they fit the tier4 criteria well. I am not sure if we can make them faster, but I have no problem with keeping them in tier3, if you are ok with that too.

File: tests/foreman/cli/test_host.py [hunk 5]
```diff
@@ -4,9 +4,9 @@
 
 :CaseAutomation: Automated
 
-:CaseLevel: Acceptance
+:CaseLevel: Component
 
-:CaseComponent: CLI
+:CaseComponent: Hammer
```
[the remark this answers is not in the document]
Comment (vsedmik) [reply tests/foreman/cli/test_host.py]: Ok, that sounds logical. I'm going to fix it.

File: tests/foreman/ui/test_host.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (mirzal) [reply tests/foreman/ui/test_host.py]: Thanks for explanation.

These times are definitely the reason to have them on `tier4`. I skimmed over tests and thought that there is nothing inherent to these tests that would make them long-running. I thought their running time is somewhere in 10-20 minutes range. But if they reach 40 minutes mark, then yeah, `tier4` sound like the right decision.

**Pull request merged**